test/e2e/generator/generate.go

```diff
@@ -149,15 +148,6 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}, upgradeVersion st
 		return manifest, fmt.Errorf("unknown topology %q", opt["topology"])
 	}
 
-	if opt["topology"].(string) == "large_partially_connected" {
-		// currently this is at max 11 and minimum 4
-		totalPossibleConnections := numSeeds + numValidators + numFulls - 1
-		// this value should be between 3 and 2
-		manifest.MaxOutboundConnections = math.MaxInt(totalPossibleConnections/3, 2)
-		// this value should be between 5 and 2
-		manifest.MaxInboundConnections = math.MaxInt(totalPossibleConnections/2, 2)
-	}
```

Comment:
not sure of the context around adding partially connected stuff, but upstream doesn't have this so it should be good to remove